src/main/java/com/epam/spring/homework2/ApplicationRunner.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.homework2;
+
+import com.epam.spring.homework2.config.FirstBeansConfig;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class ApplicationRunner {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext applicationContext =
+                new AnnotationConfigApplicationContext(FirstBeansConfig.class);
+
+        for (String beanDefinitionName : applicationContext.getBeanDefinitionNames()) {
+            System.out.println(beanDefinitionName);
```

Comment:
Task #12 implementation is still missing 
```suggestion
            System.out.println("Bean name: " + beanDefinitionName);
            System.out.println("Bean definition: " + applicationContext.getBeanDefinition(beanDefinitionName));
```